### ISSUE TITLE
[Quest] Digging Through Bones

### ISSUE THREAD

**Issue by Mewmewww** [Atlantiss/NetherwingBugtracker#2240]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)

**Description**: 
Escort Quest
**Current behaviour**:
5 seconds during the trip with the 5 dwarfs, we encounter 2 skeletons and 1 treant we kill. After they die, only the chief helps me with while the rest just stands still. They do move everytime we engage something, but they do not help us in killing enemies. Also, they are stacking up on each other.
**Expected behaviour**: 
They should keep their formations, they should help us kill our enemies.
**Server Revision**: 
2244

**Comment by mhdemn** [comment]
Would like to update with how the [quest](https://tbc-twinhead.twinstar.cz/?quest=10922) currently works in 2255:

[Chief Archaeologist Letoll](https://tbc-twinhead.twinstar.cz/?npc=22458) is standing next to the bonfire without any of his [companions](https://tbc-twinhead.twinstar.cz/?npc=22464). They are in the digging site, stacked on top of each other, and they will not respond or do any of their speeches that respond to what Letoll says throughout the quest. 

They won't help fighting the mobs either, nor will Letoll.

The only one doing it's correct pathing during the quest is Letoll. The researchers should follow him around, forming around him, helping him dig with their picks.

Also, the drum the diggers find doesn't actually appear.

Afterwards, when the event in the digging site ends, Letoll should stand idle for a little bit, tell us all to return to the camp, and then start running towards the camp along with his researchers. Currently he starts running full speed without notice, does the speech line he should have said before starting to run (not after starting), and none of the researchers follow him, they stay stacked on top of each other in the digging site.

Finally, when Letoll reaches the camp, quest gets completed, and he dies instantly as soon as he gets to his position next to the bonfire. He has a notably long respawn too, been preparing this update for 20 minutes and his body hasn't even despawned yet. 

Here's a [video](https://youtu.be/EuOjINKMiDI) with some of what I could catch as soon as I noticed it was working unproperly, and a [video](https://www.youtube.com/watch?v=0FrYTT3Ufo0) showing how it should work (it's retail footage on a later xpac, but I compared the first videos youtube show and it seems it remains unchanged since TBC).

**Comment by Mikik3jr** [comment]
Yeah I just did the quest and it was all the same as mhdemn described.

**Comment by Mikik3jr** [comment]
Some pictures I made during the escort:
![wowscrnshot_110618_135717](https://user-images.githubusercontent.com/43505031/48066177-29c32400-e1cd-11e8-9f67-e9d27e250f93.jpg)
![wowscrnshot_110618_135916](https://user-images.githubusercontent.com/43505031/48066187-35164f80-e1cd-11e8-9ac1-94c92f8ae839.jpg)
![wowscrnshot_110618_140113](https://user-images.githubusercontent.com/43505031/48066191-36477c80-e1cd-11e8-8329-b1e202890fe1.jpg)
![wowscrnshot_110618_140155](https://user-images.githubusercontent.com/43505031/48066192-36e01300-e1cd-11e8-9ce5-0e8c9a4a29f4.jpg)
![wowscrnshot_110618_140214](https://user-images.githubusercontent.com/43505031/48066194-3778a980-e1cd-11e8-8d88-2fe83513ce77.jpg)
![wowscrnshot_110618_140231](https://user-images.githubusercontent.com/43505031/48066195-38114000-e1cd-11e8-87ba-55bbe0f0e9a1.jpg)
![wowscrnshot_110618_140259](https://user-images.githubusercontent.com/43505031/48066197-38a9d680-e1cd-11e8-8f2f-7f062edd68ca.jpg)
![wowscrnshot_110618_140512](https://user-images.githubusercontent.com/43505031/48066199-38a9d680-e1cd-11e8-9632-bf848a364bec.jpg)

**Comment by Tenoutoften** [comment]
Every time you get out of combat the buffs you give to [[NPC]Chief Archaeologist Letoll#22458](https://www.wowhead.com/npc=22458/chief-archaeologist-letoll) reset. Made a video of that [here](https://www.youtube.com/watch?v=BrcAMHMMhsg&feature=youtu.be)

**Comment by KredesX** [comment]
Yep still bugged. Doable though.

**Comment by Minimix186** [comment]
Would like to add that this quest is stilled buggy on the Kara PTR. It is completeable, but the dwarfs that follow the main quest NPC, will only follow when engaged in combat.

Also the mob they spawn is invisible

https://i.imgur.com/7LcdQol.jpg

**Comment by Minimix186** [comment]
The quest you get after completing this quest, Fumping is also not working correctly.

The quest item you use "Fumper" (31810), is a casting item, which should be interrupted by movement and it isn't. You can run around while the spell continues to cast on your original position.

The worms you spawn Mature Bone Sifter, are invisible.

However, the quest is completeable.

**Comment by The14me** [comment]
The Item part is right(atm u can move till channeling the item), but I see the worms 
https://imgur.com/a/7HIVXpP
Rev.3451

**Comment by Wolffenstein** [comment]
Escort fixed in rev 3474. "Fumping" quest is a separate issue.
